Design/Packaging/Future_packaging_strategy.md

```diff
@@ -0,0 +1,70 @@
+
+- On CentOS/RHEL7, we create a package containing only our compiled program in a .rpm format. We rely on the currently available system packages, optional packages from `rhel-7-workstation-optional-rpms` and some packages from EPEL to run Mantid. We install the Mantid .rpm package using the rpm command.
+
+## Intended move forward
+
+All operating systems should have a fully built and deployable conda-build implementation, where a use can in theory run a simple command in conda and it will install Mantid with all it's runtime dependencies, which with some small tweaks we can then package on all Operating Systems assuming that the conda-build recipe has been created and ran successfully.
+
+The aim at this time is to create similar packages to what we have now on MacOSX and Windows, whilst shipping all dependencies in our Ubuntu and CentOS/RHEL7 which is a change. We will make a .exe for Windows, .dmg for MacOSX, .deb for Ubuntu, and a .rpm for CentOS/RHEL7 much like we do now on all operating systems.
+
+### Conda build
+
+We intend to distribute a conda package using the conda-build system, which will allow advanced users to install Mantid using conda. Users will be able to install this packaged version with, 
+```bash
+conda install -c mantid mantid-workbench
+```
+The user will then be able to interact with the package in a manner familiar to any users of other conda packages, e.g. Spyder, Numpy,
+```bash
+conda install -c mantid mantid-workbench // Install conda package
+python -c "import mantid"  // Import mantid
+MantidWorkbench // Launch workbench
+```
+The runtime requirement specified by the conda build package will be used to produce our offline installers, as described below. 
+
+### Producing Offline (None-Conda) packages
+
+On all operating systems we will need to create the directory from which the package will be made, the command to do this will look something like this:
+```bash
+conda create -p package_dir mantid-workbench --copy -c mantid -c conda-forge -y
+```
+
+In the case of a package that was not uploaded to the Anaconda Cloud like the previous command expects, you can do this from a locally constructed package and achieve the same result using this command:
+```bash
+conda create -p package_dir --use-local mantid-workbench --copy -c conda-forge -y
+```
+
+For Linux operating systems I suggest using FPM (`Effin Package Management - https://github.com/jordansissel/fpm) due to it's simplicity and ease of creating .deb and .rpm installers, from one machine, therefore in the future it should be possible to create all Linux distribution packages from one machine. At present it is not planned to use FPM for MacOSX as it doesn't support .dmg app bundles, and the support it does provide for .pkg files on MacOSX is less than ideal for the same reasons we already migrated away from .pkg files in the first place.
```

Comment:
I there might be a backtick missing here?

Reply:
That tick itself shouldn't have been there, it has been removed thanks for spotting this!